tests/integration/ha/test_backups.py

```diff
@@ -574,9 +574,6 @@ async def test_restore_to_new_cluster(
     """
     app = (await app_name(ops_test) or APP_NAME) if deploy_type == "small" else "main"
     logging.info("Destroying the application")
-    for machine in ops_test.model.state.machines.values():
-        # Needed due to canonical/opensearch-operator#243
-        await machine.destroy(force=True)
```

Comment:
Hey @reneradoi, thanks for that.

Can you make sure backup tests pass? It seems most failed b/c of self-hosted access. However, one of the tests: "aws-small" should've passed.

Reply:
Hi @phvalguima , in an earlier run, all backup tests were successfull: https://github.com/canonical/opensearch-operator/actions/runs/9315395684.